README.md

```diff
@@ -26,15 +22,49 @@ Pass `wrednetdbg=0` to disable the NetDbg functionality.
 
 The WhateverRed kext is licensed under BSD-3.
 
-This project is currently under active research and development. It is not functional, but in a close state to.
+This project is under active research and development and is not yet functional but close to being so. Its goal is to enable AMD iGPU support for Renoir/Raven(2) and their derivatives, such as Picasso. However, due to the complexity of GPU drivers, especially those without public source code, extended support for non-existent logic is not possible. As a result, you must use Catalina (minimum) to Big Sur (recommended) since some required iGPU logic has been removed in Monterey.
+
+Injecting kernel extensions is not possible during the OpenCore injection stage.
+
+
+## **Introduction**
+
+We are a team of three working on getting graphics acceleration for AMD iGPUs (Raven/Raven2/Renoir and their derivatives) on hackintoshes (PCs running macOS). Currently, we are fixing the missing and broken logic by patching the existing kexts (AMDRadeonX5000 for GCN 5 and AMDRadeonX6000 for VCN).
```

Comment:
The second part is incorrect. Drop it, or add the missing kexts mentioned in the original Telegram message, and remove the word "currently"

Reply:
```suggestion
We are a team of three working on getting graphics acceleration for AMD iGPUs (Raven/Raven2/Renoir and their derivatives) on hackintoshes. We are patching the existing kexts in order to achieve this (mixing AMDRadeonX5000 for GCN 5 and AMDRadeonX6000 for VCN 1/2, and replacing AMDRadeonX6000Framebuffer for DCN instead of AMD10000Controller which is DCE).
```